packages/salesforce-adapter/test/client.test.ts

```diff
@@ -544,12 +551,21 @@ describe('salesforce client', () => {
             mockOrganizationQueryResult({ orgType: NON_PRODUCTION_ORGANIZATION_TYPE, isSandbox: true })
           })
           it('should return isProduction false and correct accountType', async () => {
-            expect(await validateCredentials(credentials, 3, connection)).toEqual({
-              accountId: '',
+            expect(await validateCredentials(sandboxCredentials, 3, connection)).toEqual({
+              accountId: 'https://url.com/',
               isProduction: false,
               accountType: NON_PRODUCTION_ORGANIZATION_TYPE,
+              extraInformation: { orgId: '', accountIdVersion: '1' },
             })
           })
+          it('should throw an error when there is no instanceUrl', async () => {
+            const mockConnection = mockClient().connection
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            mockConnection.instanceUrl = undefined
```

Comment:
```suggestion
            mockConnection.instanceUrl = undefined
            _.set(mockConnection, 'instanceUrl', undefined)
```
Nit: You can consider this instead